WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -87,6 +101,10 @@ - (instancetype)init
                                                  selector:@selector(defaultAccountDidChange:)
                                                      name:WPAccountDefaultWordPressComAccountChangedNotification
                                                    object:nil];
+
+        // Watch for application badge number changes
+        NSString *badgeKeyPath = NSStringFromSelector(@selector(applicationIconBadgeNumber));
```

Comment:
Per our discussion with @diegoreymendez we decided to use a constant for the keypath:

```
static NSString * const WPApplicationIconBadgeNumberKeyPath = @"applicationIconBadgeNumber";
```